enos/modules/build_boundary_docker_local/build.sh

```diff
@@ -8,7 +8,14 @@ set -eux -o pipefail
 root_dir="$(git rev-parse --show-toplevel)"
 pushd "${root_dir}" > /dev/null
 
+# make docker image
 export IMAGE_TAG_DEV="${IMAGE_NAME}"
 make build-ui docker-build-dev
 
+# make the cli to be used by the test runner
+export GOOS=linux
+export GOARCH=arm64
```

Comment:
Will this work on amd64 machines?

Reply:
fair point. have an idea

Reply:
Don't even need that. I think I just needed the `GOOS` line